clang/test/Preprocessor/nonportable-include-trailing-whitespace.c

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: case-insensitive-filesystem
```

Comment:
Hmm, I think Mac's qualifies as a case-insensitive-filesystem, but I don't think it ignores trailing spaces/dots like Windows does(?).

Reply:
I wasn't sure about this but I think I could remove that.
The behavior should be as follows:
- Linux: You get the new warning and a file not found error.
- Windows: You only get the new warning as the file is still found.
- Mac: Warning + error just as on Linux. Case sensitivity shouldn't matter in both cases.